### ISSUE TITLE
Instruction (commands or script) how to make PR to odoo repo from a patch and vice versa

### ISSUE THREAD

**Issue by yelizariev** [xoes/odoo-patches#1]

**Comment by blaggacao** [comment]
- [x] How to generate a patch
- [x] How to apply a patch

I think both are somewhat covered in the Readme in a basic way.

Could we close this, and probably reopen other for future improvement / guides?

**Comment by yelizariev** [comment]
The idea is generate a patch from PR's url to odoo repository, .e.g.
    
     gen-odoo-patch https://github.com/odoo/odoo/pull/24816

 At this moment you need to specify reference to commit, path and you need to squash commits manually.

Oppositre idea is to create a PR to odoo from odoo-patch. We will need this when people get tired to be refused with their PR to odoo. So, they could send a patch here and then just execute a command to notify odoo about possible updates. It would also  promote the project if we put a link *Made from the patch at **xoe-labs/odoo-patches** project* in the PRs

**Comment by blaggacao** [comment]
Right, now I remember! Thanks! It seems rather an easy task. I might take that in a free minute...
